docs/mcp/configuring-mcp-servers.mdx

```diff
@@ -160,7 +205,7 @@ Example: "Analyze the performance of my API" might use an MCP tool that tests AP
 
 Common issues and solutions:
 
--   **Server Not Responding:** Check if the server process is running and verify network connectivity
--   **Permission Errors:** Ensure proper API keys and credentials are configured in your `mcp_settings.json` file
--   **Tool Not Available:** Confirm the server is properly implementing the tool and it's not disabled in settings
--   **Slow Performance:** Try adjusting the network timeout value for the specific MCP server
```

Comment:
Note this change was made by the linter as tests were failing in CI due to prettier warnings.